internal/config/config.yaml

```diff
@@ -114,3 +114,15 @@ secrets:
 billing:
   tenant_id: ""
   environment_id: ""
+
+s3:
+  enabled: true
+  region: "ap-south-1"
+  invoice:
+    bucket: "flexprice-invoices"
```

Comment:
Please keep the default config to have `enabled: false` and this can be an additional config that people will set up in case they want to use.

Reply:
Also add about all this in the SETUP.md as proper docs

Reply:
And in the docs also mention that when you set this to true - you will have to set up a new S3 bucket and allow the infra where flexprice is running to have access to push to this S3 (ex in AWS ECS we have TaskRole where we can specify this)

Reply:
From changes I can see we will need - PutObject, GetObject,HeadObject permissions